src/ducks/transactions/Transactions.jsx

```diff
@@ -58,16 +50,14 @@ const TableTrDesktop = compose(translate(), withDispatch, withUpdateCategory())(
     isExtraLarge,
     showCategoryChoice,
     filteringOnAccount,
+    selectTransaction,
     ...props
   }) => {
     const categoryId = getCategoryId(transaction)
     const categoryName = getCategoryName(categoryId)
     const categoryTitle = t(`Data.subcategories.${categoryName}`)
     const parentCategory = getParentCategory(categoryId)
-    const onSelect = () => {}
-    const onSelectDisabled = () => {
-      showComingSoon(t)
-    }
+    const onSelectTransaction = () => selectTransaction(transaction)
```

Comment:
It is better not to create function in renders as it prevents receivers of the function to bypass rendering if the function has not changed. You might want to change TableTrDesktop to a PureComponent and this to a method of the class.

> The problem with this syntax is that a different callback is created each time the LoggingButton renders. In most cases, this is fine. However, if this callback is passed as a prop to lower components, those components might do an extra re-rendering. We generally recommend binding in the constructor or using the class fields syntax, to avoid this sort of performance problem.

https://reactjs.org/docs/handling-events.html

Reply:
:+1: